### PR TITLE
Changelog for 0.8.0 Beta 1

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,76 @@
...
+* Run test suite against MongoDB 3.6 [DBZ-529](https://issues.jboss.org/browse/DBZ-529)
+* Provide option to flatten structs in MongoDB unwrapping SMT [DBZ-561](https://issues.jboss.org/browse/DBZ-561)
+* Allow configuration option for keep alive interval for Mysql binlog reader [DBZ-670](https://issues.jboss.org/browse/DBZ-670)
+* Add support for databases with encodings other than UTF-8/16/32 [DBZ-676](https://issues.jboss.org/browse/DBZ-676)
+* Provide option to specify statements to be executed upon connection creation (e.g.  connection wait timeout) [DBZ-693](https://issues.jboss.org/browse/DBZ-693)
+
+
+### Breaking changes since 0.7.5
+
+Apache kafka was upgraded to version 1.1 ([DBZ-687](https://issues.jboss.org/browse/DBZ-687)).
```
Comment (gunnarmorling) [inline CHANGELOG.md]: s/kafka/Kafka

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,76 @@
...
+Please see [upgrade documentation](http://kafka.apache.org/11/documentation.html#upgrade) for correct upgrade procedure.
+
+Topic names for heartbeat messages followed a hard-coded naming schema.
+The rules were made more flexible in [DBZ-668](https://issues.jboss.org/browse/DBZ-668).
+
+Transaction id for PostgreSQL was originnally encoded as an 32-bit `integer` type. The real range is a 64-bit `long` type so this was changed in [DBZ-673](https://issues.jboss.org/browse/DBZ-673).
```
Comment (gunnarmorling) [inline CHANGELOG.md]: s/originnally/originally/

Would also be good to mention the exact field concerned.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,76 @@
...
+
+The datatypes with timezone were not correctly offseted for databases running in non-UTC timezones. This was fixed in [DBZ-587](https://issues.jboss.org/browse/DBZ-578) and [DBZ-741](https://issues.jboss.org/browse/DBZ-741).
```
Comment (gunnarmorling) [inline CHANGELOG.md]: s/with/without/?
s/offseted/offsetted/

I'd suggest to add a link to the description of the behavior in the connector docs which I added yesterday.

**Comment by jpechane** [comment]
@gunnarmorling Changes done

**Comment by gunnarmorling** [comment]
Cool, can you merge it?

**Pull request merged**